lighthouse-core/report/v2/report-renderer.js

```diff
@@ -98,19 +102,42 @@ window.ReportRenderer = class ReportRenderer {
    * @param {string} description
    * @return {!Element}
    */
-  _renderScore(score, title, description) {
-    const element = this._createElement('div', 'lighthouse-score');
+  _renderScore(score, title, description, auditOrCategory) {
```

Comment:
Score could also be an object with value/precision props. Or maybe`type` for the latter. Examples:

```
category.score = {value: 100, precision: 'numeric'}
audit.score = {value: true, precision: 'binary'}
audit.score = {value: 'A', precision: 'grade'}
```